django/template/engine.py

```diff
@@ -107,8 +107,7 @@ def get_template_loaders(self, template_loaders):
 
     def find_template_loader(self, loader):
         if isinstance(loader, (tuple, list)):
-            args = list(loader[1:])
-            loader = loader[0]
+            loader, *args = loader
```

Comment:
I :heart: this pattern - much clearer. Also note that this works for plucking the tail or both the head and tail too:
```python
In [1]: head, *tail = [1, 2, 3]

In [2]: head, tail
Out[2]: (1, [2, 3])

In [3]: *head, tail = [1, 2, 3]

In [4]: head, tail
Out[4]: ([1, 2], 3)

In [5]: head, *body, tail = [1, 2, 3]

In [6]: head, body, tail
Out[6]: (1, [2], 3)
```
So potentially use of `[:-1]` and `[-1]` or `[-1:]` could be improved?